CoreFoundation/String.subproj/CFString.c

```diff
@@ -3233,7 +3233,7 @@ Boolean CFStringFindWithOptionsAndLocale(CFStringRef string, CFStringRef stringT
 
                                     do {
                                         str1Char = CFStringGetCharacterFromInlineBuffer(&inlineBuf1, --index);
-                                    } while (CFUniCharIsMemberOfBitmap(str1Char, graphemeBMP), (rangeToSearch.location < index));
+                                    } while (CFUniCharIsMemberOfBitmap(str1Char, graphemeBMP) && (rangeToSearch.location < index));
```

Comment:
@spevans / @phausler I think the original code is correct. The comma operator in C evaluates the first expression, discards the result and then evaluates the second and the overall expression evaluates to that (the value of the second expr).